### PR TITLE
Group Greetings

### PR DESCRIPTION
**Pull request by JPGulo**
John Gulon and Jakob Linscheid greetings.
CoAuth: TheGreenN1nja <linsc235@morris.umn.edu>

### CODE REVIEW

**Comment by thegreenn1nja** [comment]
Very nice, I like

**Review by thegreenn1nja — APPROVED** [review]
I like very nice

**Review by amoak017 — DISMISSED** [review]

**Review by thegreenn1nja — DISMISSED** [review]
looks good

**Review by Bonesbhbhbh — DISMISSED** [review]

**Review by Bonesbhbhbh — APPROVED** [review]

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -9,12 +9,22 @@ public class Hellos {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(jakobSaysHello());
```
Comment (JosieBarber) [inline src/main/java/hellos/Hellos.java:13]: J Names :0

**Review by JosieBarber — APPROVED** [review]

**Pull request merged**